src/prepare_dlc_dev_environment.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import logging
+import sys
+import toml
+
+from config import get_dlc_developer_config_path
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+# LOGGER.addHandler(logging.StreamHandler(sys.stderr))
+
+
+def get_args():
+    """
+    Manage arguments to this script when called directly
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--partner_toml",
+        default=get_dlc_developer_config_path(),
+        help="TOML file with partner developer information",
+    )
+    parser.add_argument(
+        "--frameworks",
+        nargs="+",
+        choices=["pytorch", "tensorflow"],
+        required=True,
+        help="ML Framework for which to prepare developer enviornment",
+    )
+    parser.add_argument(
+        "--job_types",
+        nargs="+",
+        choices=["training", "inference"],
+        default=["training", "inference"],
+        help="Training and inference containers to prepare developer environment",
+    )
+
+    return parser.parse_args()
+
+
+class TomlOverrider:
+    def __init__(self):
+        self._overrides = {"build": {}}
+
+    def set_build_frameworks(self, frameworks):
+        """
+        Write a function that, given a list of frameworks, assembles a dictionary with key/value pairs:
```

Comment:
Can you update this docstring so that it includes the description of the method now, rather than the instructions?